src/check-c-linkage-decls.py

```diff
@@ -23,4 +23,4 @@
 		print ('Ouch, file %s has HB_BEGIN_DECLS / HB_END_DECLS, but it shouldn\'t' % x)
 		stat = 1
 
-sys.exit (stat)
+exit (stat)
```

Comment:
https://stackoverflow.com/questions/6501121/difference-between-exit-and-sys-exit-in-python

Reply:
oh, so let's normalize in the reverse way